### PR TITLE
Modify and delete products

### PR DESCRIPTION
**Pull request by fxdotnet**
Modify and Delete functionality implemented in this feature.

### CODE REVIEW

File: PTC/Controllers/BaseController.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using System.Web;
+using System.Web.Mvc;
+
+namespace PTC.Controllers
+{
+    public abstract class BaseController : Controller
+    {
+        protected BaseController(ITrainingProductManager tpmgr)
+        {
+            _tpmanager = tpmgr;
```
Comment (MeApproveAnything) [inline PTC/Controllers/BaseController.cs]: please write out toiletpaper instead of tp

**Review by MeApproveAnything — CHANGES_REQUESTED** [review]
check it loco

File: PTC/Controllers/BaseController.cs (hunk 1 above)
Comment (fxdotnet) [reply PTC/Controllers/BaseController.cs]: No can do Beavis..

**Review by MeApproveAnything — APPROVED** [review]
good.......(lord)

**Pull request merged**